android/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
         minSdkVersion 16
         targetSdkVersion 31
         versionCode 1
-        versionName '3.5.10'
+        versionName '3.5.11'
```

Comment:
We should probably set these both to `3.5.12` to keep it consistent with iOS (And watching what's going on with Android, I think there's going to be another release in the next day or two anyway)